src/common/IPC/CommonSyscalls.h

```diff
@@ -246,14 +246,14 @@ namespace VM {
 
     enum EngineFileSystemMessages {
         FS_INITIALIZE,
-        FS_HOMEPATH_OPENMODE,
+        FS_HOMEPATH_OPENMODE, // TODO(0.52) remove
```

Comment:
I prefer the `TODO:` syntax

Reply:
I'm using `TODO(0.52)` as a search string to find stuff that I want to delete later. Do you have an idea what I should use instead?

Reply:
I usually look for the complete `TODO:` string so I’m not very happy with the missing colon, but that's not very important: those TODO are not expected to last. Make sure the related commit deleting this ends in 0.52 branch and it's OK. :grin: